backend/SnackOverflow/src/main/java/ro/utcn/sd/vasi/SnackOverflow/controller/rest/VoteController.java

```diff
@@ -0,0 +1,30 @@
+package ro.utcn.sd.vasi.SnackOverflow.controller.rest;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RestController;
+import ro.utcn.sd.vasi.SnackOverflow.services.AnswerManagementService;
+import ro.utcn.sd.vasi.SnackOverflow.services.QuestionManagementService;
+import ro.utcn.sd.vasi.SnackOverflow.services.UserManagementService;
+
+@RestController
+@RequiredArgsConstructor
+public class VoteController {
+    private final AnswerManagementService answerManagementService;
+    private final QuestionManagementService questionManagementService;
+    private final UserManagementService userManagementService;
+
+    //is put the appropiate method here?
+    @PostMapping("/questions/{questionId}/answers/{answerId}/votes/{voteValue}")
```

Comment:
Debatable... POST is fine in my view. I don't like that you pass the value in the URL though. You should pass it in the body.